docker-compose-local.yml

```diff
@@ -55,12 +54,14 @@ services:
       - MAX_FILE_UPLOAD_REQUEST_SIZE_MEGABYTES=360
       - DARTS_INBOUND_STORAGE_SAS_URL
       - DARTS_UNSTRUCTURED_STORAGE_SAS_URL
+      - JAVA_TOOL_OPTIONS=-agentlib:jdwp=transport=dt_socket,address=*:8003,server=y,suspend=n
```

Comment:
Add a docker debug port. Allows us to debug into the docker container using intellij